deploy/charts/external-secrets/templates/webhook-deployment.yaml

```diff
@@ -51,6 +51,7 @@ spec:
           - --dns-name={{ include "external-secrets.fullname" . }}-webhook.{{ .Release.Namespace }}.svc
           - --cert-dir={{ .Values.webhook.certDir }}
           - --check-interval={{ .Values.webhook.certCheckInterval }}
+          - --healthz-addr={{ .Values.webhook.readinessProbe.port }}
```

Comment:
The `--healthz-addr` is expected to be a address/port combination like `:8081`. 
I guess it would be great to limit it to localhost, but as far as i can tell we can not listen on both ipv4+ipv6 localhost with a single listen addr :thinking:.
When using `:8081` it will listen on both ipv4+ipv6.

I don't mind leaving it `:8081` as a default for users to support all networks - i'd argue it's up to the security requirements of the user to change the listen addr appropriately..

Reply:
Hi @moolen thanks for the review, I really missed that it's an address, not a port.

I agree and it is probably less cognitive load to leave the address as ":" by default.

would it be ok if the default address is ":" with the explanation as 

` # default to : for both ipv4 and ipv6`